app/views.py

```diff
@@ -289,12 +289,25 @@ def finalize_or_reject(request, process_id):
     user = request.data["authorized"]
     user_type = request.data["user_type"]
     state = request.data["action"]
+
+    if state == "rejected":
+        message = request.data.get("rejectionMsg", None)
+        if not message:
+            return Response(
+                "provide a reason for rejecting the document",
+                status = status.HTTP_400_BAD_REQUEST
+            )
+    else:
```

Comment:
Instead of this block first assign the message to an empty string before hand.

Reply:
Assigned the empty message string before the block